Fase-1/API/self-order-management/src/main/kotlin/com/fiap/selfordermanagement/adapters/driven/persistence/mapper/OrderMapper.kt

```diff
@@ -1,12 +1,24 @@
 package com.fiap.selfordermanagement.adapters.driven.persistence.mapper
 
 import com.fiap.selfordermanagement.adapters.driven.persistence.entities.OrderEntity
+import com.fiap.selfordermanagement.adapters.driven.persistence.entities.ProductEntity
 import com.fiap.selfordermanagement.application.domain.entities.Order
+import com.fiap.selfordermanagement.application.domain.entities.OrderItem
 import org.mapstruct.Mapper
+import org.mapstruct.Mapping
 
 @Mapper
```

Comment:
```suggestion
@Mapper(
    nullValueMappingStrategy = NullValueMappingStrategy.RETURN_DEFAULT, 
    nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.SET_TO_DEFAULT
)
```
Tentei com essas anotações, mas o programa não compilou